modules/oe_content_project/config/install/field.field.node.oe_project.oe_project_eu_contrib.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_project_eu_contrib
+    - node.type.oe_project
+id: node.oe_project.oe_project_eu_contrib
+field_name: oe_project_eu_contrib
+entity_type: node
+bundle: oe_project
+label: 'EU contribution'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  min: null
+  max: null
```

Comment:
Probably have to limit the max value, otherwise if in some case user adds a number like `1234567890123456789` we will not have any form error but fatal error (on trying to save node). It is reproducible on EWPP side.


`Drupal\Core\Database\DatabaseExceptionWrapper: SQLSTATE[22003]: Numeric value out of range: 1264 Out of range value for column 'oe_project_eu_budget_value' at row 1: INSERT INTO "node__oe_project_eu_budget" ("entity_id", "revision_id", "bundle", "delta", "langcode", "oe_project_eu_budget_value") VALUES (:db_insert_placeholder_0, :db_insert_placeholder_1, :db_insert_placeholder_2, :db_insert_placeholder_3, :db_insert_placeholder_4, :db_insert_placeholder_5); Array ( [:db_insert_placeholder_0] => 12 [:db_insert_placeholder_1] => 20 [:db_insert_placeholder_2] => oe_project [:db_insert_placeholder_3] => 0 [:db_insert_placeholder_4] => en [:db_insert_placeholder_5] => 1.2345678901235E+18 ) in Drupal\Core\Entity\Sql\SqlContentEntityStorage->saveToDedicatedTables() (line 1393 of /var/www/html/build/core/lib/Drupal/Core/Entity/Sql/SqlContentEntityStorage.php).`